cornice/service.py

```diff
@@ -508,11 +508,14 @@ def cors_support_credentials_for(self, method=None):
         return False
 
     def cors_max_age_for(self, method=None):
+        max_age = None
         for meth, view, args in self.definitions:
             if method and meth.upper() == method.upper():
-                return args.get('cors_max_age', False)
+                max_age = args.get('cors_max_age', None)
```

Comment:
we should break here to avoid continue the for loop. Only the first value matters.